tools/sidekick/config.json

```diff
@@ -42,7 +42,7 @@
       "id": "localize-2",
       "title": "Localize (V2)",
       "environments": [ "edit" ],
-      "url": "https://main--bacom--adobecom.hlx.page/tools/loc?milolibs=locui",
+      "url": "https://main--bacom--adobecom.aem.page/tools/loc?milolibs=locui",
```

Comment:
Do these need to be da-?
https://main--da-bacom--adobecom.aem.page/tools/loc?milolibs=locui

Reply:
Indeed! Fixed